crop_classification/inference.py

```diff
@@ -0,0 +1,116 @@
+from cropharvest.engineer import Engineer
+from datetime import datetime
+from pathlib import Path
+import numpy as np
+import xarray as xr
+import pandas as pd
+import re
+import torch
+
+from typing import Dict, List, Optional, Tuple, Union
+
+
+class Inference:
+    """
+    Class for running inference using either a pytorch checkpoint model or
+    a pytorch jit model on a single tif file.
+    """
+
+    def __init__(
+        self, model, device: Optional[torch.device] = None, batch_size: int = 64
+    ):
+        self.model = model
+        self.device = device
+
+        try:
+            self.normalizing_dict: Dict[str, np.ndarray] = {
+                k: np.array(v) for k, v in self.model.normalizing_dict_jit.items()
+            }
+        except AttributeError:
+            self.normalizing_dict: Dict[str, np.ndarray] = None
```

Comment:
Re normalizing dict, let me know if something like this makes sense @gabrieltseng